retrospective-update/functions.py

```
@@ -0,0 +1,1065 @@
+                       credentials: str,
+                       CL: CloudLog) -> None:
+    CL.ping('RUNNING', 'syncing-finalstates-to-S3')
+    result = subprocess.run(
+        f's5cmd '
+        f'--credentials-file {credentials} --profile odp '
+        f'sync --include "*finalstate*" --size-only '
+        f'{outputs_dir}/ '
+        f'{s3_qfinal_dir}/',
+        shell=True, capture_output=True, text=True,
+    )
+    if not result.returncode == 0:
```

Ruff (0.12.2)

Use `result.returncode != 0` instead of `not result.returncode == 0`

Replace with `!=` operator

(SIM201)